relayer/src/types.ts

```diff
@@ -0,0 +1,48 @@
+import type { BaseRelayerService } from "./services/types";
```

Comment:
Hm the naming of this file sounds a bit odd to me. A lot of logic is happening here, so I would not say it is just declaring types. Can't we just call it `relayer-program.ts`?

Reply:
✅